src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c

```diff
@@ -419,10 +418,15 @@ static uint16_t dcd_ep_ctr_handler(void)
           uint8_t userMemBuf[8];
           /* Get SETUP Packet*/
           count = pcd_get_ep_rx_cnt(USB, EPindex);
-          //TU_ASSERT_ERR(count == 8);
-          dcd_read_packet_memory(userMemBuf, *pcd_ep_rx_address_ptr(USB,EPindex), 8);
+          if(count == 8) // Setup packet should always be 8 bits. If not, ignore it, and try again.
```

Comment:
I guess you mean 8 bytes not bits. Le'ts me know when you think PR is ready for merging.

Reply:
Yep, 64 bits. Fixed.